data/json/recipes/recipe_electronics.json

```diff
@@ -2239,5 +2239,33 @@
     [ [ "power_supply", 1 ] ],
     [ [ "cable", 5 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "vac_sealer",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_TOOLS",
+  "skill_used": "electronics",
+  "skills_required": [ "mechanics", 2 ],
+  "difficulty": 4,
+  "time": 40000,
+  "reversible": true,
+  "autolearn": false,
+  "book_learn": [[ "advanced_electronics", 4 ] , [ "mag_electronics", 6 ] , [ "manual_electronics", 5 ]],
+  "qualities" : [
+    {"id":"SCREW","level":1,"amount":1}
+  ], "components": [
+    [
+      [ "scrap", 4 ]
+    ],
+    [
+      [ "element", 2 ]
+    ],
+    [
+      [ "hose", 1 ]
+    ],
+    [
+      [ "power_supply", 1 ]
```

Comment:
Agreed, either dump the craftable pneumatic weapons or let the vacuum sealer in.